README.md

```diff
@@ -176,7 +176,28 @@ npm install ./aws-clickstream-web-x.x.x.tgz
 ```
 Note: Please correct the SDK version and change the path to where the `aws-clickstream-web-x.x.x.tgz` file is located.
 
-**Test**
+### Integrate by `clickstream-web.min.js` file
+Execute the following same script to generate `clickstream-web.min.js` which will be located in the `/dist` folder.
+```bash
+cd clickstream-web && npm run pack
+```
+Copy the `clickstream-web.min.js` into your project and add the following initial code into your `index.html`.
+
+```html
+<script src="clickstream-web.min.js"></script>
```

Comment:
Add `defer` in script tag?

Reply:
This will affect the initialize of the `clickstream-web.min.js` script. And it is not recommended to use defer to load our SDK after the document is parsed.

Reply:
I see other scripts supporting `async` attribute. How do we improve the script loading performance in client environments?

https://github.com/chipzoller/hugo-clarity/blob/master/layouts/partials/analytics.html

Reply:
Currently user can add `async` and put the initialize code after `DOMContentLoaded` like:
```html
 <script async src="clickstream-web.js"></script>
 <script>
   document.addEventListener("DOMContentLoaded", function () {
     window.ClickstreamAnalytics.init({
       appId: 'your appId',
       endpoint: 'http://example.com/collect',
     })
   });
 </script>
```
For more convenient to lazy load our SDK. we need to change the SDK initialize process. And planned to test and Implement it in the next version.